### PR TITLE
Homepage created 

### PR DESCRIPTION
**Pull request by archit7448**
### What to check 
- [x] Reducer.jsx 
- [x] note.jsx 
- [x] all jsx files 
 [check this files](https://github.com/archit7448/note-taking-app/pull/1/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true)
### What not to check 
- [ ] CSS files 
- [ ] Svg files 
- [ ] package.json and modules

### CODE REVIEW

File: src/Pages/SignIn/signIn.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <input
+              type="text"
+              name="password"
+              placeholder="password"
+              className="form-input"
+              value={password}
+              onChange={(event) => setPassword(event.target.value)}
+            />
+          </label>
+          {/* {error && <h1>something went wrong</h1>} */}
```
Comment (zoef-yt) [inline src/Pages/SignIn/signIn.jsx:50]: remove commented code.

File: src/Pages/SignIn/signIn.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          <button
+            className="button button-secondary button-form"
+            onClick={(event) => {
+              event.preventDefault();
+              loginHandler({
+                email: "adarshbalika@gmail.com",
+                password: "adarshBalika123",
+              });
+            }}
```
Comment (zoef-yt) [inline src/Pages/SignIn/signIn.jsx:59]: make it a function and call the function here.

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks great. Only suggestion in UI is  to either disable button when the text field are empty so I should not be able to submit without any fields

File: src/Pages/SignIn/signIn.jsx (hunk 1 above)
Comment (archit7448) [reply src/Pages/SignIn/signIn.jsx:50]: done

**Review by zoyatanweer — APPROVED** [review]
code is clean
easy to understand
good to merge!

**Pull request merged**